run-build-functions.sh

```diff
@@ -323,6 +332,24 @@ install_dependencies() {
       echo "Error installing pip dependencies"
       exit 1
     fi
+  elif [ -f Pipfile ]
+  then
+    echo "Installing dependencies from Pipfile"
+    if $HOME/python$PIPENV_RUNTIME/bin/pipenv install
```

Comment:
might be useful to set `PIPENV_VENV_IN_PROJECT` as multiple venvs are not needed on Netlify builds.  setting that might improve performance a little.

Reply:
Good point, thanks! 🥇

Reply:
I think we should move this (and the `export` lines from above) to the `Dockerfile` if possible. I'd rather `pipenv` be installed already, rather than done every build on-demand.